core/templates/dev/head/components/skills-mastery-list/skills-mastery-list.directive.html

```diff
@@ -0,0 +1,107 @@
+<div class="oppia-skills-list-container">
+  <md-card class="oppia-page-card oppia-long-text">
+    <div class="skills">
+      <h3 class="form-heading"> Skills included in this topic </h3>
+      <h4 ng-if="!$ctrl.userIsLoggedIn" style="color: gray;"> Please login to view your skill mastery. </h4>
+      <table class="skill-mastery-table" ng-if="$ctrl.skillIdsAndMastery.length > 0">
+        <colgroup>
+          <col style="width: 40%;">
+          <col style="width: 60%">
+        </colgroup>
+        <tr>
+          <th class="skill-mastery-table-headings">
+            Skill Descriptions
+          </th>
+          <th class="skill-mastery-table-headings">
+            Skill Mastery
+          </th>
+        </tr>
+        <tr ng-repeat="item in $ctrl.skillIdsAndMastery | orderBy: '-mastery'"
+            class="list-item"
+            style="height: 4em;"
+            ng-click="$ctrl.openConceptCardModal(item.skillId)">
+          <td> <[$ctrl.getSkillDescriptions()[item.skillId]]> </td>
+          <td>
+            <div class="oppia-skill-mastery-container">
+              <div class="score-bar" style="width: <[$ctrl.getMasteryPercentage($ctrl.getDegreesOfMastery()[item.skillId])]>%; background: <[$ctrl.getColorForMastery($ctrl.getDegreesOfMastery()[item.skillId])]>"></div>
+            </div>
+          </td>
+        </tr>
+      </table>
+      <h4 ng-if="$ctrl.skillIdsAndMastery.length === 0" style="color: gray;">
+        No skill is assigned to this topic.
+      </h4>
+    </div>
+  </md-card>
+</div>
+
+<style>
+skills-mastery-list .oppia-page-card {
+  margin-top: 2%;
+  margin-left: 10%;
```

Comment:
Arrange in alphabetic order. (Ditto elsewhere.)

Reply:
Done.